src/app/components/elements/panels/EventOverviews.tsx

```diff
@@ -27,84 +27,240 @@ export const EventOverviews = ({setSelectedEventId, user}: {user: PotentialUser;
-                <RS.Table bordered className="mb-0 bg-white">
+        <ShowLoading
+          until={eventOverviews}
+          thenRender={(eventOverviews) => (
+            <React.Fragment>
+              {atLeastOne(eventOverviews.length) && (
+                <div className="overflow-auto">
+                  <RS.Table
+                    bordered
+                    className="mb-0 bg-white"
+                    style={{ maxWidth: "100%" }}
+                  >
                     <thead>
-                        <tr>
-                            <th className="align-middle text-center">
-                                Actions
-                            </th>
-                            <th className="align-middle"><RS.Button color="link" onClick={() => {setSortPredicate('title'); setReverse(!reverse);}}>
-                                Title
-                            </RS.Button></th>
-                            <th className="align-middle"><RS.Button color="link" onClick={() => {setSortPredicate('date'); setReverse(!reverse);}}>
-                                Date
-                            </RS.Button></th>
-                            <th className="align-middle"><RS.Button color="link" onClick={() => {setSortPredicate('bookingDeadline'); setReverse(!reverse);}}>
-                                Booking deadline
-                            </RS.Button></th>
-                            <th className="align-middle"><RS.Button color="link" onClick={() => {setSortPredicate('location.address.town'); setReverse(!reverse);}}>
-                                Location
-                            </RS.Button></th>
-                            <th className="align-middle"><RS.Button color="link" onClick={() => {setSortPredicate('eventStatus'); setReverse(!reverse);}}>
-                                Status
-                            </RS.Button></th>
-                            <th className="align-middle"><RS.Button color="link" onClick={() => {setSortPredicate('numberOfConfirmedBookings'); setReverse(!reverse);}}>
-                                Number confirmed
-                            </RS.Button></th>
-                            <th className="align-middle"><RS.Button color="link" onClick={() => {setSortPredicate('numberOfWaitingListBookings'); setReverse(!reverse);}}>
-                                Number waiting
-                            </RS.Button></th>
-                            <th className="align-middle"><RS.Button color="link" onClick={() => {setSortPredicate('numberAttended'); setReverse(!reverse);}}>
-                                Number attended
-                            </RS.Button></th>
-                        </tr>
+                      <tr>
+                        <th className="align-middle text-center">Actions</th>
+                        <th className="align-middle">
+                          <RS.Button
```

Comment:
All these buttons seem to be exactly the same, except for the button text and the sort predicate. Can't they be moved to its own component (inside this file)?